holidays/countries/finland.py

```diff
@@ -97,6 +102,83 @@ def _populate_public_holidays(self):
         # Second Day of Christmas.
         self._add_christmas_day_two(tr("Tapaninpäivä"))
 
+    def _populate_unofficial_holidays(self):
+        if self._year >= 1854:
+            # Runeberg Day
+            self._add_holiday_feb_5(tr("Runebergin päivä"))
+
+        if self._year >= 1860:
+            # Kalevala Day, Day of Finnish Culture
+            self._add_holiday_feb_28(tr("Kalevalan päivä, suomalaisen kulttuurin päivä"))
+
+        if self._year >= 2007:
+            #  Minna Canth Day, Day of Equality
+            self._add_holiday_mar_19(tr("Minna Canthin päivä, tasa-arvon päivä"))
+
+        if self._year >= 1978:
+            # Mikael Agricola Day, Day of the Finnish Language
+            self._add_holiday_apr_9(tr("Mikael Agricolan päivä, suomen kielen päivä"))
+
+        if self._year >= 1987:
+            # National War Veterans' Day
+            self._add_holiday_apr_27(tr("Kansallinen veteraanipäivä"))
+
+        if self._year >= 2019:
+            # Europe Day
+            self._add_holiday_may_9(tr("Eurooppa-päivä"))
+
+        if self._year >= 1918:
+            # Mothers' Day
+            self._add_holiday_2nd_sun_of_may(tr("Äitienpäivä"))
+
+        if self._year >= 1952:
+            # J. V. Snellman Day, Day of Finnish Heritage
+            self._add_holiday_may_12(tr("J.V. Snellmanin päivä, suomalaisuuden päivä"))
+
+        if self._year >= 1977:
+            # Remembrance Day
+            self._add_holiday_3rd_sun_of_may(tr("Kaatuneitten muistopäivä"))
+
+        if self._year >= 1942:
+            # Flag Day of the Finnish Defence Forces
```

Comment:
We normally use en_US
```suggestion
            # Flag Day of the Finnish Defense Forces
```

Reply:
The official name is with "c" though: https://en.wikipedia.org/wiki/Finnish_Defence_Forces
But I can change this of course.